src/pytom_tm/entry_points.py

```diff
@@ -568,38 +576,24 @@ def match_template(argv=None):
     parser = argparse.ArgumentParser(
         description="Run template matching. -- Marten Chaillet (@McHaillet)"
     )
-    parser.add_argument(
+    io_group = parser.add_argument_group("Template, search volume and output")
```

Comment:
oxford comma reads nicer in my opinion
```suggestion
    io_group = parser.add_argument_group("Template, search volume, and output")
```